### PR TITLE
fix: translations on Team and other components

### PR DESCRIPTION
**Pull request by juzhiyuan**
fix https://github.com/hoppscotch/hoppscotch/issues/1761

### CODE REVIEW

File: lang/zh-CN.json [hunk 1]
```diff
@@ -304,5 +304,9 @@
...
   "confirm": "确认",
   "new_version_found": "检测到新版本，刷新就会自动更新哟。",
-  "size": "大小"
-}
+  "size": "大小",
+  "team_text_owner": "拥有者",
+  "team_text_editor": "编辑",
+  "team_text_viewer": "仅查看",
+  "select_team": "选择团队"
+}
```
Comment (juzhiyuan) [inline lang/zh-CN.json:312]: It seems that the pre-commit hook will remove the last line 🤔

Comment (liyasthomas) [reply lang/zh-CN.json:312]: That's not an issue.

**Comment by brageryz** [comment]
@juzhiyuan, I have provided **only a few examples**. There are more not translated texts. The following examples are not all - I think so.

---

https://github.com/hoppscotch/hoppscotch/blob/d1118933027286962be53cb67ed59f5439361399/components/environments/index.vue#L10
https://github.com/hoppscotch/hoppscotch/blob/d1118933027286962be53cb67ed59f5439361399/components/smart/ColorModePicker.vue#L8
https://github.com/hoppscotch/hoppscotch/blob/d1118933027286962be53cb67ed59f5439361399/components/smart/AccentModePicker.vue#L3

**Comment by liyasthomas** [comment]
With the release of v2 - this PR is now absolute.

**Pull request closed without merging**